subsys/shell/Kconfig.backends

```diff
@@ -27,8 +27,9 @@ if SHELL_BACKEND_SERIAL
 # Internal config to enable UART interrupts if supported.
```

Comment:
Typo in commit msg, s/Recntly/Recently/, better would be to say here "Recently this feature was accidentally turned off in PR #11556"

Reply:
fixed